src/ostree.jsx

```diff
@@ -677,17 +690,9 @@ class Application extends React.Component {
                 <PageSection>
                     {this.state.error && <Alert variant="danger" isInline title={this.state.error} />}
                     <Card id="deployments">
-                        <CardHeader>
-                            <CardTitle><Text component={TextVariants.h2}>{_("Deployments and updates")}</Text></CardTitle>
-                            <CardActions>
-                                <Button variant="secondary"
-                                        id="check-for-updates-btn"
-                                        isLoading={!!client.local_running || !!this.state.progressMsg}
-                                        isDisabled={!!client.local_running || !!this.state.progressMsg}
-                                        onClick={this.checkForUpgrades}>
-                                    {_("Check for updates")}
-                                </Button>
-                            </CardActions>
+                        <CardHeader actions={{ actions: actionButton, hasNoOffset: false }}>
+                            <CardTitle component="h2">{_("Deployments and updates")}</CardTitle>
+
```

Comment:
Uneeded new line (I know it's added by the script)

Reply:
Fixed.